Arguments needed to smoothly build modules without further config by the user

Details:
 - ibus: fails due to wrong UCD directory
 - gst-plugins-bad: might try to enable vulkan extension when some very common
                    packages are installed, but vulkan-headers is not.

diff --git a/jhbuild/defaults.jhbuildrc b/jhbuild/defaults.jhbuildrc
index bb211464..c94d8f04 100644
--- a/jhbuild/defaults.jhbuildrc
+++ b/jhbuild/defaults.jhbuildrc
@@ -91,7 +91,11 @@ repos = {}
 cvsroots = {}
 svnroots = {}
 branches = {}
-module_autogenargs = {}
+module_autogenargs = {
+    'ibus': '--with-ucd-dir=/usr/share/unicode/',
+}
 module_cmakeargs = {}
-module_mesonargs = {}
+module_mesonargs = {
+    'gst-plugins-bad': '-D vulkan=disabled',
+}
 module_makeargs = {}
